README.md

```diff
@@ -16,6 +16,9 @@
 <a href="https://scitools-iris.readthedocs.io/en/latest/?badge=latest">
 <img src="https://readthedocs.org/projects/scitools-iris/badge/?version=latest"
      alt="Documentation Status" /></a>
+<a src="https://results.pre-commit.ci/latest/github/SciTools/iris/master">
```

Comment:
I think this should he a `href` not `src`.  The `README.md` on your forked repo branch just links to the image, not the pre commit url: https://github.com/bjlittle/iris/tree/add-pre-commit-ci-badge

Reply:
:blush: